userspace/falco/app_cmdline_options.cpp

```diff
@@ -155,7 +155,7 @@ void cmdline_options::define()
 #else
 		("c",                             "Configuration file. If not specified tries " FALCO_SOURCE_CONF_FILE ", " FALCO_INSTALL_CONF_FILE ".", cxxopts::value(conf_filename), "<path>")
 #endif
-		("A",                             "Monitor all events, including those with EF_DROP_SIMPLE_CONS flag.", cxxopts::value(all_events)->default_value("false"))
+		("A",                             "Monitor all events, including not interesting ones. Please use the `--i` command line option to see the ignored events. This option has no effect if the capture is not live", cxxopts::value(all_events)->default_value("false"))
```

Comment:
```suggestion
		("A",                             "Monitor all events, including not interesting ones. Please use the `-i` command line option to see the ignored events. This option has no effect if the capture is not live", cxxopts::value(all_events)->default_value("false"))
```

Reply:
:facepalm: great catch!

Reply:
I'd change the last sentence with `This option is implicit when the capture is not live.` or something similar.